<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit { border-radius: 5px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="addStockFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="10,10,10,10,10,10" columnstretch="30,30" columnminimumwidth="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="spacing">
         <number>20</number>
        </property>
        <item row="4" column="1">
         <widget class="QLineEdit" name="marketValueLineEdit"/>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="priceLabel">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="sharesLabel">
          <property name="text">
           <string>Shares</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="costBasisLabel">
          <property name="text">
           <string>Cost Basis</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="priceLineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item row="5" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="gainLabel">
          <property name="text">
           <string>Gain</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignRight">
         <widget class="QLabel" name="marketValueLabel">
          <property name="text">
           <string>Market Value</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="sharesLineEdit">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="costBasisLineEdit"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="gainLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="buttonFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton{
background-color: #008B8B;
       border: 1px solid #c9c9c9;
       border-radius: 3px;
       padding: 5px;
       margin-right: 5px;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
           </property>
           <property name="centerButtons">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
